src/pages/components/progress-indicator/usage.mdx

```diff
@@ -45,41 +48,208 @@ set of steps, guiding toward the completion of a specified process.
 
 ## Overview
 
-Use progress indicators to keep the user on track when completing a specific
-task. By dividing the end goal into smaller, sub-tasks, it increases the
-percentage of completeness as each task is completed.
+Progress indicators help manage a user's expectations when completing a
+multistep process. They show which step the user is currently on, the total
+number of steps, and overall progress in completing a task.
 
-## Best practices
+<Row>
+<Column colLg={8}>
+
+![Example of progress indicator used in a payment UI](images/progress-indicator-usage-1.png)
+
+</Column>
+</Row>
+
+### When to use
+
+- When the user is working through a linear process that can be organized into
+  three or more steps.
+- When the user could benefit from understanding their progress on long forms
+  such as eCommerce checkouts, onboarding, or visa applications.
+- When user inputs should be validated before progressing to the next step.
+- To complement standard back/next navigation in a linear sequence.
+
+### When not to use
 
-#### Logical progression
+- When a process or form has fewer than three steps.
+- When the process may be completed in any order.
+- When the number of steps may change based on conditional logic.
 
-Display the steps in order from left to right. Indicate to the user that they
-are performing a multistep process, and show the direction of movement. Allow
-the user to return to a previous step to review their data submission.
+### Variants
 
-#### Indicate the current step
```

Comment:
Probably need to check in with devs but I don't think Variant table is needed here. This could be explained under the Behaviors section. Refer to [this example in Menu](https://carbondesignsystem.com/components/menu/usage/#interactions)

Reply:
Looking at this example, I agree! I aupdated the content under 'Mouse' and also updated the language about variants in the component description.